### PR TITLE
Fix extension build instructions

### PR DESCRIPTION
**Pull request by markmhendrickson**
The instructions for building the extension from source are incorrect since `yarn` is instructed for installation inside of the package directory instead of the root directory. This PR fixes that issue while simplifying the build commands using npm scripts.

### CODE REVIEW

File: packages/app/README.md [hunk 1]
```diff
@@ -50,8 +50,11 @@ Then, open [localhost:8080](http://localhost:8080).
 Clone this repository, install dependencies and build the extension:
 
 ~~~bash
-git clone https://github.com/blockstack/ux && cd ux && yarn
-cd packages/app && yarn lerna run build --scope @stacks/connect-ui && yarn lerna run prod:ext
+git clone https://github.com/blockstack/ux
+cd ux
+yarn
+yarn run build:connect-ui
+yarn run build:extension
```
Comment (hstove) [inline packages/app/README.md]: `yarn run` is not needed - any script can be used. This is different than NPM btw, where you have to specify `run` for all non-standard scripts.

So, just `yarn build:connect-ui` and `yarn build:extension` work!

**Review by hstove — CHANGES_REQUESTED** [review]
I'm not sure why the `yarn.lock` has so many changes - that shouldn't happen for a PR with no dependency changes.

**Comment by markmhendrickson** [comment]
Should I nuke the yarn.lock and regenerate?

**Review by hstove — APPROVED** [review]
I updated to:

- Have `yarn build:connect-ui` run after running `yarn` inside the monorepo. This should help reduce the commonality of people running into the "unbuilt connect-ui" problem.
- Fixed yarn.lock stuff
- Updated Playwright - I had updated in Connect 2.0 to fix CI stuff, so this is the same thing
- Changed the README command to just be `yarn build:extension`

Thanks @markmhx for putting this together!

**Pull request merged**